webapp/src/app/services/phaser/entities/entity-manager.ts

```diff
@@ -177,6 +179,10 @@ export class EntityManager extends BaseObject {
 						const p = {x: pointer.worldX, y: pointer.worldY};
 						if (this.leftClickOpts.timer < 200 && Vec2.distance2(p, this.leftClickOpts.pos) < 10) {
 							this.selectEntity(entity, this.multiSelectKey.isDown);
+
+							if (!this.multiSelectKey.isDown && this.leftClickOpts.prevTimer < 500) {
+								entity.doubleClick();
```

Comment:
can you add the proper type to entity, so it's easier for the ide to find this function usage